packages/fury-adapter-oas3-parser/lib/parser/parseReference.js

```diff
@@ -8,7 +8,19 @@ function isReferenceObject(element) {
 
 function parseReference(component, parser, context, element) {
   if (isReferenceObject(element)) {
-    return parseReferenceObject(context, component, element);
+    const result = parseReferenceObject(context, component, element);
```

Comment:
Took me a moment to realize that this isn't a smelly variable name, but a variable for "parse result". Perhaps `parseResult` would cognitively lead better to realizing what's in it?

Reply:
I'll prepare a separate PR changing the variable name, this is a frequent pattern in most files. I'll sed the code base and do it all.